client/flutter/lib/pages/about_page.dart

```diff
@@ -26,6 +26,7 @@ class AboutPage extends StatelessWidget {
               color: Colors.white,
               padding: EdgeInsets.symmetric(horizontal: 25, vertical: 10),
               child: RichText(
+                textScaleFactor: MediaQuery.textScaleFactorOf(context),
```

Comment:
A better fix for this is using `Text.rich` instead of `RichText` (the Flutter API is really confusing here)

Reply:
haha, I was none the wiser.